### DISCUSSION TITLE
DIY CNC GCode Embroidery Machine

### DISCUSSION THREAD

**Discussion by Blackbumpkin** [inkstitch/inkstitch#1581] [Q&A]
Hello,
I'm Blackbumpkin and I want to build a CNC GCode Embroidery Machine for my daughters. The tutorial I found was the incentive.
[https://inkstitch.org/tutorials/embroidery-machine/](https://inkstitch.org/tutorials/embroidery-machine/)

Now I will only have one question about my project that I have to solve, so I am looking for a solution to get answers and at the same time I would like to document the project and make it available to other users if there is interest.  Since I'm new to the world of GitHub and not good at English either, it's hard for me to see if this is the right way to do it?

Does it make sense to put it here?  If YES, how and where do  i start it best.

Greetings Blackbumpkin

**Comment by UrticaDesign2** [comment]
We are building the CNC DIY embroidery sewing machine too, but it stopped when we came to the firmware for the stepper motors. The solution described at the Inkstitch webpage. What did you do in these steps?

**Reply by X3msnake** [reply]
What is your questions regarding the firmware?
The project uses a universal CNC firmware that is called GRBL that and can be configured for any hardware.

What electronics hardware do you have or are planning on aquiring
